### DISCUSSION TITLE
Migrate to new installation

### DISCUSSION THREAD

**Discussion by icehuntman** [akhilrex/hammond#102] [Q&A] [answered]
Hello all,

I want to migrate to a fresh installation. Is there a possibility to migrate data from an existing installation to a new one?
I'm using a container for the current and new installation.

**Comment by zenjubbers** [comment, marked as answer]
I simply copied the directory I have mapped to /config to the new install. I think all the data is in /config/hammond.db and as long as you copy that to your new install, you should get all your data back in the new install.
